Mathlib/LinearAlgebra/Projectivization/Action.lean

```diff
@@ -0,0 +1,45 @@
+/-
+Copyright (c) 2025 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+import Mathlib.LinearAlgebra.Projectivization.Basic
+import Mathlib.GroupTheory.GroupAction.Ring
+
+/-!
+# Group actions on projectivization
+
+Show that (among other groups), the general linear group of `V` acts on `ℙ K V`.
+-/
+
+open scoped LinearAlgebra.Projectivization Matrix
+
+namespace Projectivization
+
+section DivisionRing
+
+variable {α K V : Type*} [AddCommGroup V] [DivisionRing K] [Module K V]
```

Comment:
Optional nitpick: it would be nice if we could call this `G`
```suggestion
variable {G K V : Type*} [AddCommGroup V] [DivisionRing K] [Module K V]
```